backend/api/Database/Context/InitDb.cs

```diff
@@ -116,6 +116,11 @@ private static List<Mission> GetMissions()
 
     public static void PopulateDb(FlotillaDbContext context)
     {
+        // To make sure we are not trying to initialize database more than once during tests
```

Comment:
I thought we had solved this

Reply:
So though I, but Xunit insists on initiating the database twice